### PR TITLE
remove QgsLegendModel

### PR DESCRIPTION
**Pull request by 3nids**

### CODE REVIEW

File: python/core/composer/qgscomposerlegend.sip [hunk 1]
```diff
@@ -60,13 +60,6 @@ class QgsComposerLegend : QgsComposerItem
      */
     bool resizeToContents() const;
 
-    /** Returns pointer to the legend model*/
-    //! @deprecated in 2.6 - use modelV2()
-    QgsLegendModel* model() /Deprecated/;
-
-    //! @note added in 2.6
-    QgsLegendModelV2* modelV2();
-
```
Comment (wonder-sk) [inline python/core/composer/qgscomposerlegend.sip]: modelV2() should stay?

**Comment by 3nids** [comment]
@wonder-sk indeed, otherwise is it ok for you to remove it?

**Comment by wonder-sk** [comment]
I would suggest to drop original model() and rename modelV2() to model()...

**Comment by 3nids** [comment]
yes that will be next step indeed!!! My question is mainly: ok to remove this from the API?

**Comment by wonder-sk** [comment]
Sorry I am a bit slow today... what do you mean with "this" ? :-)

**Comment by 3nids** [comment]
No problem, I might be not that clear either ;)
I mean throwing away QgsLegendModel.
Since it came with the old legend, I thought we should remove it.

**Comment by wonder-sk** [comment]
Ah right, yes the old QgsLegendModel should go...

**Comment by 3nids** [comment]
:)

**Pull request merged**